data/alexandria start.txt

```diff
@@ -0,0 +1,1290 @@
+			`	"The fourth book tells of HoD Qav's travel across the world aboard a giant ship that can cross oceans. The merchant gathers vast resources and builds an armada while seeing sights nobody in Botolphstone had ever dreamed of.`
+			`	"More struggles across the world begin fifth book as the chaos escalates and expands. HoD Qav helps resolve disputes, defend the weak, and steal from the rich. The world descends into chaos with only one merchant holding back the inevitable."`
+			choice
+				`	"What happens next?"`
+				`	(Change the subject.)`
+					goto questions
+			`	"Next? How can I know what happens next? Only five of the eight books were ever written."`
+			`	"There are rumors that the writer of these books is an ancient being of great power over many worlds. He may still be alive, and writing the later books. Nobody has seen or heard from him in centuries though. He may even be a myth."`
+				goto questions
+			label services
+			`	"It's a library, fool. What do you think we do? Serve bacon? Teach juggling lessons?`
+			`	"We have books. Lots of books. If you want to read books, come here to our library."`
+			label "library services"
+			choice
+				`	"Bacon?"`
+					goto bacon
+				`	"Juggling lessons?"`
+					goto juggling
+				`	"Books?"`
+					goto "library books"
+				`	"Change the subject."`
+					goto questions
+				`	"End the conversation."`
+					goto end
+			label bacon
+			`	"It was a joke. If you want bacon, go look in the Dirt Belt. There's plenty of food out there."`
+				goto "library services"
+			label juggling
+			`	"Are you daft? We don't provide juggling lessons here. Find such silliness on the Paradise Planets."`
+				goto "library services"
+			label "library books"
+			`	"Yes. We have the largest archive in all of Human space. If you're looking for a book you will find it here."`
+				goto "library services"
+			label jobs
+			`	"Yes.`
+			`	"The library will pay you to Seek for rare books or artwork and bring them here for archival.`
+			`	"We'll also pay you to ferry librarians or library materials between Alexandria Station and other libraries. Find those jobs in Job Boards here or elsewhere.`
+			`	"The library has a mandate to preserve forbidden knowledge. If you can reach pirate planets, you can find jobs to smuggle illegal literature to Alexandria.`
+			`	"Once you've proven yourself, we may give you more difficult jobs: a book cruise, long-range escort missions, or hunting book thieves."`
+				goto questions
+			label end
+			`"Excellent. Welcome to your new life as an Alexandria Seeker."`
+	on decline
+		fail "Access to Alexandria"
+	on fail
+		fail "Access to Alexandria"
+
+mission "Access to Alexandria"
+	name "Access to Alexandria"
+	description "Access to Alexandria"
+	repeat 2
```

Comment:
Why `repeat`? (and even if it does have repeat, why exactly twice?)

Reply:
Sometimes the mission vanished once. This is my workaround. If it repeats endlessly, you end up with an infinite number of "Access to Alexandria" missions in your save file.